src/gui/midi.cpp

```diff
@@ -97,6 +97,22 @@ MidiHandler Midi_none;
 
 #endif
 
+struct DB_Midi {
+	uint8_t status;
+	size_t cmd_len;
+	size_t cmd_pos;
```

Comment:
> using `size_t` to replace `Bitu` is quite safe.

Absolutely; we also know that these variables are perfectly safe as 32-bit `uint`'s because the "official" builds of DOSBox are 32-bit.